### DISCUSSION TITLE
Downloads complete but it only gets 7kb of video diles

### DISCUSSION THREAD

**Discussion by scoobydad** [aclap-dev/video-downloadhelper#438] [Help and Q&A]
### Your question

The plugin and companion app are updated. I have  purchased both licenses.
When I try to download videos lately from p*rnhub it gets kbs of the files most of the time it used to get the whole video.
I am on Window 10, tried it under Firefox and Vivaldi (Chromium) and same issue. Any reason?

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\ericc",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I am running Windows 11 Pro 64-bit version 23H2 build 22631.3447, Firefox 125.0.2 64-bit, licensed VDH 8.2.2.8a1 beta, CoApp 2.0.19.  I do not work for VDH.  I'm just another user.

Over the years, folks have reported problems with Pornhub.  It's always worked for me.  Well, almost always.  I have encountered odd behavior where something will fail one time & a couple of hours later it works fine.  There seems to be no predicting.  But in general, it seems to present no problems for VDH.

But there may be a problem getting the same beta as I have.  That is the latest beta.  The problem is that the link on the Wiki for the beta is perhaps not correct.  It was not correct a few weeks ago.  I have not checked lately so I don't know if it's been fixed.  In any case, the instructions I posted here #391 should work no matter what.

GitHub is not Google.  Post a URL if you want others to see if they have the same problem with a particular video that you do.

**Reply by scoobydad** [reply]
Same thing happens in Vivaldi (chromium). The app and plug in are updated. This time I had more security off.
I will see if I can get the beta

**Comment by Season4episode10** [comment]
As another said "this can be random, may be a server side thing, maybe not, sometimes you'll find 2 or more options almost the same size on some sites, one will download and one (others) won't, you can use the data you've gleaned (which one succeeded and which one failed) to make your choices on that site during that session, but don't expect the same behavior next time on the same site, sometimes there's a bunch of download options displayed but one or sometimes more than one are valid video downloads that mirror the streaming video, others are not ... some might be an add with the same name as the wanted video, some might be some other unwanted crap, sometimes those will fail (often) making you think the correct video failed, learn to differentiate the different data displayed and your choice will be more likely to succeed in a complete "wanted" download, you'll get to know what to do the more you use the extension if you take greater notice of what's displayed by the extension, sometimes guessing and patience are your only option.
